tests/unittest/llmapi/test_llm_utils.py

```
@@ -58,3 +62,25 @@ def test_LlmArgs_default_gpus_per_node():
+
+    # put data to queue sync in a thread
+    # async get data from queue in the current event loop
+    # NOTE: the event loop in the two threads are different
+
+    def put_data_to_queue():
+        for i in range(10):
+            time.sleep(0.1)
+            queue.put(i)
+
+    async def get_data_from_queue():
+        for i in range(10):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)